cernopendata/modules/fixtures/data/docs/cms-guide-xsec/cms-guide-xsec.md

```diff
@@ -0,0 +1,201 @@
+    <tr><td>ttbarZ        </td><td>NLO     </td><td>--                  </td><td>NLO </td><td>Inclusive  </td><td>0.2057     </td><td>+0.019 -0.024    </td></tr>
+    <tr><td>tqZ; q!=b     </td><td>aMC@NLO </td><td>m(ll) > 50 GeV      </td><td>NLO </td><td>Z->leptons </td><td>0.02450    </td><td>+3.3% -2.6%      </td></tr>
+    <tr><td>tbZ           </td><td>aMC@NLO </td><td>m(ll) > 50 GeV      </td><td>NLO </td><td>Z->leptons </td><td>0.0114     </td><td>+3.3% -2.6%      </td></tr>
+    <tr><td>WWW           </td><td>aMC@NLO </td><td>--                  </td><td>NLO </td><td>Inclusive  </td><td>8.058e-02  </td><td>+4.7% -3.9%      </td></tr>
+    <tr><td>WWZ           </td><td>aMC@NLO </td><td>--                  </td><td>NLO </td><td>Inclusive  </td><td>5.795e-02  </td><td>+5.6% -4.6%      </td></tr>
+    <tr><td>WZZ           </td><td>aMC@NLO </td><td>--                  </td><td>NLO </td><td>Inclusive  </td><td>1.968e-02  </td><td>+6.0% -4.9%      </td></tr>
+    <tr><td>ZZZ           </td><td>aMC@NLO </td><td>--                  </td><td>NLO </td><td>Inclusive  </td><td>5.527e-03  </td><td>+2.7% -2.4%      </td></tr>
+    <tr><td>tttt          </td><td>aMC@NLO </td><td>--                  </td><td>NLO </td><td>Inclusive  </td><td>9.144e-04  </td><td>+36.3%, -27.0%   </td></tr>
+    <tr><td>W+ W-         </td><td>MCFM 6.6</td><td>0                   </td><td>NLO </td><td>W->eν W->eν</td><td>0.6472     </td><td>±0.0231 (±0.0266)</td></tr>
+    <tr><td>W+ Z/a*       </td><td>MCFM 6.6</td><td>m(l+l-) > 12 GeV    </td><td>NLO </td><td>W->μν Z->ee</td><td>0.0748     </td><td>±0.0025 (±0.0029)</td></tr>
+    <tr><td>W+ Z/a*       </td><td>MCFM 6.6</td><td>m(l+l-) > 40 GeV    </td><td>NLO </td><td>W->μν Z->ee</td><td>0.0535     </td><td>±0.0018 (±0.0028)</td></tr>
+    <tr><td>W- Z/a*       </td><td>MCFM 6.6</td><td>m(l+l-) > 12 GeV    </td><td>NLO </td><td>W->μν Z->ee</td><td>0.0446     </td><td>±0.0021 (±0.0018)</td></tr>
+    <tr><td>W- Z/a*       </td><td>MCFM 6.6</td><td>m(l+l-) > 40 GeV    </td><td>NLO </td><td>W->μν Z->ee</td><td>0.0305     </td><td>±0.0014 (±0.0014)</td></tr>
+    <tr><td>Z/a* Z/a*     </td><td>MCFM 6.6</td><td>both dileptonic <br>m(l+l-) > 12 GeV </td><td>NLO </td><td>Z->μμ Z->ee   </td><td>0.0385 </td><td>±0.0011 (±0.0011)</td></tr>
+    <tr><td>Z/a* Z/a*     </td><td>MCFM 6.6</td><td>both dileptonic <br>m(l+l-) > 40 GeV </td><td>NLO </td><td>Z->μμ Z->ee   </td><td>0.0185 </td><td>±0.0007 (±0.0007)</td></tr>
+    <tr><td>Z/a* Z        </td><td>MCFM 6.6</td><td>m(l+l- from Z/a*) > 12 GeV </td><td>NLO   </td><td>Z->ee Z->νν </td><td>0.1318       </td><td>±0.0040 (±0.0067)</td></tr>
+    <tr><td>Z Z           </td><td>MCFM 6.6</td><td>both dileptonic <br>m(l+l-) > 1 GeV  </td><td>NLO   </td><td>Z->ee Z->μμ </td><td>0.0173 </td><td>±0.0067 (±0.0007)</td></tr>
+  </tbody>
+</table>
+<br>
+
+## <a name="run2">Run 2</a>
```

Comment:
If readers come to this page from the Run2 dataset cross-section information, we could point them directly here and consider guiding them explicitly on the cross-section values to be used.
We could either explain generator-level cross sections vs NLO values in the dataset cross-section information text (#3581) or start this section by saying that these are not necessarily the values they want to use.

Reply:
I've added an update for this but was just a little too slow to catch it in this PR. Will come in the next!